src/borg/helpers.py

```diff
@@ -818,12 +818,34 @@ def SortBySpec(text):
     return text.replace('timestamp', 'ts')
 
 
+# Not too rarely, we get crappy timestamps from the fs, that overflow some computations.
+# As they are crap anyway, nothing is lost if we just clamp them to the max valid value.
+# msgpack can only pack uint64. datetime is limited to year 9999.
```

Comment:
ok, but assuming we are on a 64bit platform, the native datatype being used is then int64, not uint64.

so, guess it is more efficient if we clamp to 0 .. 2**63 - 1?